### PR TITLE
Some facebook user dont have email address

### PR DESCRIPTION
**Pull request by sushrest**
Some facebook user dont have email address so added facebook identifier email address instead

### CODE REVIEW

**Comment by SocalNick** [comment]
@sushrest I don' think we can trust users without email being verified. Also, this PR has conflicts.

File: src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php [hunk 1]
```diff
@@ -319,6 +319,12 @@ protected function instantiateLocalUser()
     protected function facebookToLocalUser($userProfile)
     {
         if (!isset($userProfile->emailVerified)) {
+            //some facebook user's dont have email address
+            if ($userProfile->email == ''){
+              $userProfile->email = $userProfile->identifier.'@facebook.com';
+              $userProfile->email = $userProfile->emailVerified;
```
Comment (SocalNick) [inline src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php]: Why set `$userProfile->email` in each of these lines? Also, you are setting it to `$userProfile->emailVerified` which we just confirmed wasn't set.

**Comment by sushrest** [comment]
@SocalNick Some users are registered with only mobile numbers in facebook in that case the emailVerified and email field is blank. I personally prefer to allow those users to my app as well. Or is there any ways I can override this function with my own namespace? would appreciate your help.

File: src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php (hunk 1 above)
Comment (sushrest) [inline src/ScnSocialAuth/Authentication/Adapter/HybridAuth.php]: Ya sorry about the typo there ... it should be $userProfile->emailVerified = $userProfile->email. As mentioned earlier some users are only registered with their mobile numbers in facebook and I wanted to allow those users to register into my app as well.

**Pull request closed without merging**